hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/ReadableTokenRelationStoreImpl.java

```diff
@@ -51,15 +51,15 @@ public ReadableTokenRelationStoreImpl(@NonNull final ReadableStates states) {
      * {@inheritDoc}
      */
     @Override
-    public Optional<TokenRelation> get(@NonNull final AccountID accountId, @NonNull final TokenID tokenId) {
+    @Nullable
+    public TokenRelation get(@NonNull final AccountID accountId, @NonNull final TokenID tokenId) {
         requireNonNull(accountId);
         requireNonNull(tokenId);
 
-        if (AccountID.DEFAULT.equals(accountId) || TokenID.DEFAULT.equals(tokenId)) return Optional.empty();
+        if (AccountID.DEFAULT.equals(accountId) || TokenID.DEFAULT.equals(tokenId)) return null;
 
-        final var tokenRelation = Objects.requireNonNull(readableTokenRelState)
+        return Objects.requireNonNull(readableTokenRelState)
```

Comment:
can we do that check directly in the constructor?

Reply:
yes it is done already. Removed